ibc-clients/cw-context/src/utils/mod.rs

```diff
@@ -0,0 +1,22 @@
+mod codec;
+
+pub use codec::*;
+use ibc_core::client::types::error::ClientError;
+use ibc_core::client::types::{Height, HeightError};
+
+/// Decodes a `Height` from a UTF-8 encoded byte array.
+pub fn parse_height(encoded_height: Vec<u8>) -> Result<Option<Height>, ClientError> {
+    let height_str = match alloc::str::from_utf8(encoded_height.as_slice()) {
+        Ok(s) => s,
+        Err(_) => return Ok(None),
```

Comment:
Can you please add the comment here too?

Reply:
[6de21ed](https://github.com/cosmos/ibc-rs/pull/1167/commits/6de21edb62137b9d74daf1bcacf3b626e45013cc)